stats/index.js

```diff
@@ -12,9 +12,8 @@ for (var j = 0; j < 100; ++j) {
   let stages = []
 
   for (var i = 1; i < 4; ++i) {
-    let utxos = Simulation.generateTxos(20 / i, min, max)
-    let txos = Simulation.generateTxos(80 / i, min, max / 3)
-
+    let utxos = Simulation.generateTxos(20 / i, min, max, {106: 0.9, 300: 0.1})
```

Comment:
It was *way* too much work for this one little constant (since the simulation is kind of artificial anyway - for example it does not simulate moving fees), but I did some analysis on 2 years of blockchain, there you go

https://gist.github.com/runn1ng/8e2881e5bb44e01748e46b3d1c549038

I will take weights from the table at "simulating-lengths" that I calculated down there